cpu/esp32/periph/rtt_hw_sys.c

```diff
@@ -74,18 +74,18 @@ static void _sys_init(void)
 static void _sys_poweron(void)
 {
     /* route all interrupt sources to the same RTT level type interrupt */
-    intr_matrix_set(PRO_CPU_NUM, TIMER_SYSTEM_INT_SRC, CPU_INUM_RTC);
+    intr_matrix_set(PRO_CPU_NUM, TIMER_SYSTEM_INT_SRC, CPU_INUM_RTT);
 
     /* set interrupt handler and enable the CPU interrupt */
-    xt_set_interrupt_handler(CPU_INUM_RTC, _sys_isr, NULL);
-    xt_ints_on(BIT(CPU_INUM_RTC));
+    xt_set_interrupt_handler(CPU_INUM_RTT, _sys_isr, NULL);
```

Comment:
For this file only the define `CPU_INUM_RTC` is renamed to keep it compilable. The changes for ESP-IDF for interrupt handling for this file are in commit 2a9ffee7610ab569d8f933765fddc52cae5b9143.